facdb/sql/nysparks_historicplaces.sql

```diff
@@ -0,0 +1,34 @@
+DROP TABLE IF EXISTS _nysparks_historicplaces;
+
+SELECT
+    uid,
+    source,
+    resource_name as facname,
+    NULL as addressnum,
+    NULL as streetname,
+    NULL as address,
+    NULL as city,
+    NULL as zipcode,
+    county as boro,
+    NULL as borocode,
+    NULL as bin,
+    NULL as bbl,
+    'State Historic Place' as factype,
+    'Historical Sites' as facsubgrp,
+    NULL as opname,
+    'The New York State Office of Parks, Recreation and Historic Preservation' as overagency,
+    NULL as opabbrev,
+    'NYSOPRHP' as overabbrev,
+    NULL as capacity,
+    NULL as captype,
+    NULL as proptype,
+    ST_POINT(longitude::double precision, latitude::double precision) as wkb_geometry,
```

Comment:
for nysparks_historicplaces, include `"GEOMETRY=AS_WKT"` in destination.options to create a formated `wkt` field

Reply:
https://github.com/NYCPlanning/db-data-library/issues/164